pywren/wrenhandler.py

```diff
@@ -303,7 +341,10 @@ def consume_stdout(stdout, queue):
             if total_runtime > job_max_runtime:
                 logger.warning("Process exceeded maximum runtime of {} sec".format(job_max_runtime))
                 # Send the signal to all the process groups
-                os.killpg(os.getpgid(process.pid), signal.SIGTERM)
+                if sys.platform.startswith('linux'):
+                    os.killpg(os.getpgid(process.pid), signal.SIGTERM)
+                else:
+                    pass
```

Comment:
I'm guessing kill is not supported in windows ?

Reply:
I think this was because process group IDs work a lot different in windows. I'll look into this.